sessions/TypeScript-100/common-types-examples.ts

```diff
@@ -0,0 +1,150 @@
+const isFun: boolean = false;
```

Comment:
Seems this file is located in the root folder. Please move it into the relevant subfolder.